src/extension.ts

```diff
@@ -236,6 +237,18 @@ export async function activateInternal(context: ExtensionContext) {
     },
   )
 
+  commands.registerCommand(
+    'affinidiExplorer.renameProject',
+    (element: BasicTreeItemWithProject) => {
+      telemetryHelpers.trackCommand('affinidiExplorer.renameProject', {
+        projectId: element.projectId,
+        resource: element.contextValue,
```

Comment:
`resource` not needed. We know its a project.

Reply:
removed